### ISSUE TITLE
[Logging v2] Windows RKE2 	rancher-logging-root-win-agent-fluentbit pod is in Imagepullbackoff

### ISSUE THREAD

**Issue by ronhorton** [rancher/rancher#38851]
**Rancher Server Setup**
- Rancher version: `2.6-head Commit ID cb9ccde`
- Installation option (Docker install/Helm Chart): Helm
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE2
- Proxy/Cert Details:

**Information about the Cluster**
- Kubernetes version: ` v1.24.4`
- Cluster Type (Local/Downstream): `downstream`
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider): `ec2`

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions: `admin`

**Describe the bug**
Installing rancher-logging:100.1.3+up3.17.7 - rancher-logging-root-win-agent-fluentbit goes into imagepullbackoff and does not recover - errimagepull; This results in the logging-crd unable to create resources. 

**To Reproduce**
1. import windows rke2 cluster
2. install rancher-logging

**Result**
fluentbit pod is in error state; 
**Expected Result**
logging installs; required pods are active; logs flowing to elasticsearch as indicated in cluster-ouput. 
**Screenshots**
<img width="1344" alt="2022-09-08_16-41-17" src="https://user-images.githubusercontent.com/89212856/189222496-ea1a8182-9f89-4a65-b7ef-09ad075d4e7d.png">

<img width="1360" alt="2022-09-08_16-44-07" src="https://user-images.githubusercontent.com/89212856/189222579-eabfb95c-1a69-4a74-a53f-81870d7f7222.png">

**Additional context**
reproducible 100%

**Comment by eliyamlevy** [comment]
Fixed issue by changing rancher/fluent-bit image to an existing version. I.e. the version was upgraded without check the image existed.

**Comment by ronhorton** [comment]
testing this in `2.7-head commit id 4092002` i was able to delete the rancher-logging-root-win-agent-fluentbit pod and when it restarted, it ran successfully. 

`

Fluent Bit v1.8.9
--
Mon, Oct 10 2022 3:01:58 pm | * Copyright (C) 2019-2021 The Fluent Bit Authors
Mon, Oct 10 2022 3:01:58 pm | * Copyright (C) 2015-2018 Treasure Data
Mon, Oct 10 2022 3:01:58 pm | * Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
Mon, Oct 10 2022 3:01:58 pm | * https://fluentbit.io
Mon, Oct 10 2022 3:01:58 pm |  
Mon, Oct 10 2022 3:02:00 pm | [2022/10/10 19:02:00] [ info] [engine] started (pid=7056)
Mon, Oct 10 2022 3:02:00 pm | [2022/10/10 19:02:00] [ info] [storage] version=1.1.5, initializing...
Mon, Oct 10 2022 3:02:00 pm | [2022/10/10 19:02:00] [ info] [storage] in-memory
Mon, Oct 10 2022 3:02:00 pm | [2022/10/10 19:02:00] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
Mon, Oct 10 2022 3:02:00 pm | [2022/10/10 19:02:00] [ info] [cmetrics] version=0.2.2
Mon, Oct 10 2022 3:02:00 pm | [2022/10/10 19:02:00] [ info] [filter:kubernetes:kubernetes.0] https=1 host=kubernetes.default.svc.cluster.local port=443
Mon, Oct 10 2022 3:02:01 pm | [2022/10/10 19:02:00] [ info] [filter:kubernetes:kubernetes.0] local POD info OK
Mon, Oct 10 2022 3:02:02 pm | [2022/10/10 19:02:02] [ info] [filter:kubernetes:kubernetes.0] testing connectivity with API server...
Mon, Oct 10 2022 3:02:03 pm | [2022/10/10 19:02:03] [ info] [filter:kubernetes:kubernetes.0] connectivity OK
Mon, Oct 10 2022 3:02:03 pm | [2022/10/10 19:02:03] [ info] [sp] stream processor started`

**This is fixed**

**Comment by prachidamle** [comment]
Confirmed with @eliyamlevy that rancher-logging chart has fixes to use the rancher/fluent-bit: 1.8.9 image for Windows, since the previous image tag `1.9.3` does not exist for windows. https://github.com/rancher/rancher/issues/38851

```
diff -uNr charts/rancher-logging/100.1.3+up3.17.7/values.yaml to-be-released/rancher-logging/101.0.0+up3.17.7/values.yaml
--- charts/rancher-logging/100.1.3+up3.17.7/values.yaml	2022-11-11 01:03:54.000000000 -0800
+++ to-be-released/rancher-logging/101.0.0+up3.17.7/values.yaml	2022-11-11 00:26:01.000000000 -0800
@@ -154,7 +154,7 @@
   nodeagent_fluentbit:
     os: "windows"
     repository: rancher/fluent-bit
-    tag: 1.9.3
+    tag: 1.8.9
   fluentbit_debug:
     repository: rancher/mirrored-fluent-fluent-bit
     tag: 1.9.3-debug
```
